config/scripts/validate-esm.js

```diff
@@ -115,6 +115,15 @@ function validateExportConditions(pointer, conditions) {
       return
     }
 
+    if (typeof relativeExportPath === 'object') {
+      validateExportConditions(
```

Comment:
Ouch! Thank you for spotting this. We didn't have any support for nested export conditions. This is nice ❇️